### ISSUE TITLE
Windows 10 legs appear to be missing

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#21195]
![image](https://user-images.githubusercontent.com/2642209/27267566-54819300-5477-11e7-94ab-6496cf327ada.png)

cc: @danmosemsft

**Comment by danmosemsft** [comment]
@mmitche stopped the runs - I don't recall why - but 4df5a40ec6e7f3f8d9f3ccdadbec2b1cc31604f8 apparently didn't remove the badges.
We're ultimately migrating to having Helix run the tests and badges generated from there.

**Comment by stephentoub** [comment]
So the badges should be removed? Where are we on replacements?

**Comment by mmitche** [comment]
@stephentoub Badges should be removed.  We haven't started on replacements yet.  Covered by: https://github.com/dotnet/core-eng/issues/992

**Comment by stephentoub** [comment]
> We haven't started on replacements yet

😦

**Comment by stephentoub** [comment]
What about the netfx badges?

**Comment by danmosemsft** [comment]
What are you looking for here @stephentoub some we have the regular runs? Is it the convenience of seeing badges?

**Comment by stephentoub** [comment]
> What are you looking for here @stephentoub some we have the regular runs? Is it the convenience of seeing badges?

I want to know the status of master at a glance.  How else can I easily determine that?

**Comment by danmosemsft** [comment]
@stephentoub right -- it's just funny that we have to do different Jenkins runs in order to show the badges. We should just have badges against the regular Helix runs so we can discard the rolling runs. @mmitche @Chrisboh any idea when you might have https://github.com/dotnet/core-eng/issues/992 up ?

**Comment by stephentoub** [comment]
> it's just funny that we have to do different Jenkins runs in order to show the badges

It doesn't really matter to me what infrastructure the badges come from, only that they represent runs against the latest source in master.

> so we can discard the rolling runs

What does this mean?  I think it's valuable that the badges represent the code currently checked in to master, which means the moment you merge, runs are initiated and the badges reflect those runs (today that means they switch to blue to indicate runs are in progress).

**Comment by danmosemsft** [comment]
@stephentoub we had a discussion on this a while back. 

In the team, almost nobody seems to look at the badges -- on a regular basis maybe just you and me. Myself I only look at the daily runs, and haven't found myself ever using the commit-list that the badges offer. Certainly a one stop place to see state that's visible to the community is valuable so I was suggesting just pointing the badges at the scheduled runs, which run about twice a day.

If you find the timeliness valuable maybe that's worth keeping. How that's achieved engineering wise is for @Chrisboh 's team to determine. Rather than having runs simply existing for the badges, perhaps it means badges pointing at Helix and making those runs happen more often and maybe include a commit list.

cc @weshaggard

**Comment by weshaggard** [comment]
I agree it would be ideal to just show the badges from the official runs instead of having yet another set of runs just for these badges.

On that same note with the recent CI changes I also enabled jobs for push triggers but they don't seem to be getting triggered yet. @mmitche see https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_uapaot+CGroup_Release+AGroup_x86+TestOuter_false/ as an example. I'm not sure why they aren't getting triggered.

**Comment by danmosemsft** [comment]
@weshaggard note that those runs currently do not (1) run continuously (2) display a list of commits since the last run. as well as (3) aren't wired up to badges. (1) and (2) are clearly good things, although I'm not sure that many people rely on them currently.

**Comment by weshaggard** [comment]
I agree they aren't currently wired up but I'm suggesting we should wire them up and decide if twice daily is enough, which I think it likely is.
